### DISCUSSION TITLE
Random blank labels in UNETR training

### DISCUSSION THREAD

**Discussion by btho733** [Project-MONAI/MONAI#6759] [Q&A]
Hi,
I'm following 3D unetr tutorial (https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/unetr_btcv_segmentation_3d.ipynb) to build a multi class segmentation pipeline.  As suggested in that notebook, training patches are generated using RandCropByPosNegLabeld transform. Out of curiosity, I displayed the patches (both image and corresponding label) that are going into training loop. 

<img width="388" alt="Blanklabel_unetr" src="https://github.com/Project-MONAI/MONAI/assets/7256608/c20a4ad6-df86-4b2e-a98a-f99f06d04017">

As shown in the screenshot above, I am expecting patches similar to the ones in top panel(see Training 48/1000). Most of the training patches follow this. But I am observing that random patches turn out with just blank labels as in the bottom panels(see Training 49/1000). It looks like these training patches are skewing my results at the moment. 

May I know why this occurs in the training ? 
Any suggestion @ahatamiz , @yiheng-wang-nv , @wyli 

Thanks
BT

**Comment by KumoLiu** [comment]
Hi @btho733, IMO it makes sense since you are using `RandCropByPosNegLabeld` if you don't want the patches with the whole blank labels, you can try to increase your patch_size. Or just use `RandCropByPosNegLabeld` with `pos=1`.
Hope it helps, thanks!
